### ISSUE TITLE
Can't install 0.22

### ISSUE THREAD

**Issue by dbuenzli** [c-cube/ocaml-containers#90]
(sequence is installed with 0.9)
```
> opam install containers
The following actions will be performed:
  ∗  install containers 0.22

=-=- Gathering sources =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[containers] Archive in cache

=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[ERROR] The compilation of containers failed at "make build".
Processing  1/1: [containers: ocamlfind remove]
#=== ERROR while installing containers.0.22 ===================================#
# opam-version 1.2.2
# os           darwin
# command      make build
# path         /Users/dbuenzli/.opam/4.03.0/build/containers.0.22
# compiler     4.03.0
# exit-code    2
# env-file     /Users/dbuenzli/.opam/4.03.0/build/containers.0.22/containers-92685-d2c37b.env
# stdout-file  /Users/dbuenzli/.opam/4.03.0/build/containers.0.22/containers-92685-d2c37b.out
# stderr-file  /Users/dbuenzli/.opam/4.03.0/build/containers.0.22/containers-92685-d2c37b.err
### stdout ###
# [...]
# /Users/dbuenzli/.opam/4.03.0/bin/ocamlfind ocamldep -package bytes -package sequence -package result -modules src/advanced/CCCat.ml > src/advanced/CCCat.ml.depends
# cppo -D 'OCAML_MAJOR 4' -D 'OCAML_MINOR 3' -o src/advanced/CCMonadIO.ml src/advanced/CCMonadIO.cppo.ml
# /Users/dbuenzli/.opam/4.03.0/bin/ocamlfind ocamldep -package bytes -package sequence -package result -modules src/advanced/CCMonadIO.ml > src/advanced/CCMonadIO.ml.depends
# /Users/dbuenzli/.opam/4.03.0/bin/ocamlfind ocamlc -c -g -annot -bin-annot -safe-string -short-paths -no-alias-deps -w A -I src/core -w -4 -w -44 -package bytes -package sequence -package result -I src/advanced -I src/core -o src/advanced/containers_advanced.cmo src/advanced/containers_advanced.ml
# /Users/dbuenzli/.opam/4.03.0/bin/ocamlfind ocamlc -c -g -annot -bin-annot -safe-string -short-paths -no-alias-deps -w A -I src/core -w -4 -w -44 -package bytes -package sequence -package result -I src/advanced -I src/core -o src/advanced/CCLinq.cmo src/advanced/CCLinq.ml
# + /Users/dbuenzli/.opam/4.03.0/bin/ocamlfind ocamlc -c -g -annot -bin-annot -safe-string -short-paths -no-alias-deps -w A -I src/core -w -4 -w -44 -package bytes -package sequence -package result -I src/advanced -I src/core -o src/advanced/CCLinq.cmo src/advanced/CCLinq.ml
# File "src/advanced/CCLinq.ml", line 215, characters 14-30:
# Error: Unbound value Sequence.flatMap
# Hint: Did you mean flat_map?
# Command exited with code 2.
### stderr ###
# E: Failure("Command ''/Users/dbuenzli/.opam/4.03.0/bin/ocamlbuild' src/core/containers.cma src/core/containers.cmxa src/core/containers.a src/core/containers.cmxs src/io/containers_io.cma src/io/containers_io.cmxa src/io/containers_io.a src/io/containers_io.cmxs src/unix/containers_unix.cma src/unix/containers_unix.cmxa src/unix/containers_unix.a src/unix/containers_unix.cmxs src/sexp/containers_sexp.cma src/sexp/containers_sexp.cmxa src/sexp/containers_sexp.a src/sexp/containers_sexp.cmxs src/data/containers_data.cma src/data/containers_data.cmxa src/data/containers_data.a src/data/containers_data.cmxs src/iter/containers_iter.cma src/iter/containers_iter.cmxa src/iter/containers_iter.a src/iter/containers_iter.cmxs src/string/containers_string.cma src/string/containers_string.cmxa src/string/containers_string.a src/string/containers_string.cmxs src/advanced/containers_advanced.cma src/advanced/containers_advanced.cmxa src/advanced/containers_advanced.a src/advanced/containers_advanced.cmxs src/bigarray/containers_bigarray.cma src/bigarray/containers_bigarray.cmxa src/bigarray/containers_bigarray.a src/bigarray/containers_bigarray.cmxs src/threads/containers_thread.cma src/threads/containers_thread.cmxa src/threads/containers_thread.a src/threads/containers_thread.cmxs src/top/containers_top.cma src/top/containers_top.cmxa src/top/containers_top.a src/top/containers_top.cmxs examples/id_sexp.native -use-ocamlfind -j 1 -tag debug' terminated with error code 10")
# make: *** [build] Error 1



=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
The following actions failed
  ∗  install containers 0.22
No changes have been performed

=-=- containers.0.22 troubleshooting =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
=> Small release of containers, making use of the new releases of
     qtest and qcheck.
```

**Comment by c-cube** [comment]
Thanks for the report. Indeed, I'm guilty of forgetting one instance of a deprecated function.

**Comment by c-cube** [comment]
Should be fixed by release 0.22.1 (https://github.com/ocaml/opam-repository/pull/8169). Sorry for the inconvenience.
